setuptools/dist.py

```diff
@@ -313,8 +313,9 @@ def check_package_data(dist, attr, value):
     for k, v in value.items():
         if not isinstance(k, six.string_types):
             raise DistutilsSetupError(
-                "{!r} must be strings (got {!r})".format(
-                    'keys of {!r} dict'.format(attr), k))
+                "keys of the {!r} dict must be strings (got {!r})"
+                .format(attr, k)
```

Comment:
I'm thinking we might as well avoid the awkward nesting quoting when possible, such that this returns `keys of 'package_data' dict must be strings (got 400)` rather than `"keys of 'package_data' dict" must be strings (got 400)`.